README.md

```diff
@@ -1,78 +1,3 @@
 # web-crash-course
 
-```mermaid
```

Comment:
Please add some additional info about the project, description seems poor

Reply:
ok boss, i'' do that in the next task